src/main/java/org/gridsuite/cases/importer/job/CaseImportLogger.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.cases.importer.job;
+
+import com.datastax.driver.core.PreparedStatement;
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.Row;
+
+import java.util.Date;
+
+import static com.datastax.driver.core.querybuilder.QueryBuilder.*;
+
+/**
+ * @author Nicolas Noir <nicolas.noir at rte-france.com>
+ */
+public class CaseImportLogger {
```

Comment:
Could be Autocloseable?

Reply:
CaseImportLogger and SftpConnection made AutoCloseable